extension/llm/modules/attention.py

```diff
@@ -177,13 +195,20 @@ def setup_cache(
                 "Key value caches are already setup. You cannot call ``setup_caches()`` twice. Skipping."
             )
         else:
-            self.kv_cache = InferenceKVCache(
-                batch_size=batch_size,
-                max_seq_len=max_seq_len,
-                num_kv_heads=self.num_kv_heads,
+            # self.kv_cache = InferenceKVCache(
```

Comment:
can you try adding `from executorch.extension.llm.custom_ops import *` here and see if that works?